7_fetalSC_vs_LGR5+_and_LGR5-/0_Grün_2016_create_anndata_file.ipynb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
<p>As above, everything needs a description.</p><p> </p><p><code>gene.coutt.csv</code>   Is this correct or a typo?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>